docs/OpenAPIExamples.rst

```diff
@@ -0,0 +1,338 @@
+====================================
+Python API Examples (Open Endpoints)
+====================================
+
+The HCA API provides several ways for users of the Human Cell Atlas (HCA) to access and download
+data sets from the HCA. This page covers how to access the HCA using Python API bindings.
+
+*NOTE:* The HCA cli utility is compatible with Python 3.5+.
+
+
+create_version()
```

Comment:
Consider adding code styling to all of these headings since they are code. If you like this idea, consider adding this to ALL such similar headings.

```suggestion
``create_version()``
```

Reply:
Done